README.Rmd

```diff
@@ -49,18 +48,20 @@ Install the latest released version of the package with:
 install.packages("epinowcast", repos = "https://epinowcast.r-universe.dev")
 ```
 
-Alternatively, install the stable development version from GitHub using the following,
+Alternatively, install the development version (whilst we strive to limit breaking changes or the introduction of bugs during development this version may contain both) from GitHub using the following,
 
 ```{r, eval = FALSE}
 remotes::install_github("epinowcast/epinowcast", dependencies = TRUE)
 ```
 
-The unstable development version can also be installed from GitHub using the following,
+Historical package releases can be installed from GitHub using `remotes` by adding the release tag as in the following example (which installs [`0.2.0`](https://github.com/epinowcast/epinowcast/releases/tag/v0.2.0)),
 
 ```{r, eval = FALSE}
-remotes::install_github("epinowcast/epinowcast@develop", dependencies = TRUE)
+remotes::install_github("epinowcast/epinowcast@v0.2.0", dependencies = TRUE)
```

Comment:
I think this version is better if people might want to conditionally use "HEAD" vs some other tag
```suggestion
remotes::install_github("epinowcast/epinowcast", dependencies = TRUE, ref = "v0.2.0")
```